hr_cae_contract_recruitment/models/hr_contract.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Manuel Claeys Bouuaert <manuel@coopiteasy.be>
+#   Robin Keunen <robin@coopiteasy.be>
+# Copyright 2020 GRAP
+#   Quentin Dupont <quentin.dupont@grap.coop>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class Contract(models.Model):
+    _inherit = "hr.contract"
+
+    turnover_minimum = fields.Monetary(string="Minimum Turn-Over")
+
+    @api.onchange("employee_id")
+    def onchange_employee_id(self):
+        super().onchange_employee_id()
+        self.turnover_minimum = self.employee_id.turnover_minimum
```

Comment:
You removed `if self.employee_id.turnover_minimum` here. I would keep it, such that when changing to an employee who doesn't have a `turnover_minimum` set yet, a `turnover_minimum` which could be set on the contract is not lost/replaced by zero.